content/renderer/media/media_factory.cc

```diff
@@ -750,10 +750,17 @@ MediaFactory::CreateRendererFactorySelector(
     is_base_renderer_factory_set = true;
 #if BUILDFLAG(USE_STARBOARD_MEDIA)
     // TODO(b/326827007): Revisit renderer to support secondary videos.
-    factory_selector->AddFactory(
-        RendererType::kStarboard,
-        std::make_unique<media::StarboardRendererFactory>(media_log));
-    factory_selector->SetBaseRendererType(RendererType::kStarboard);
+    if (media::IsStarboardRendererEnabled()) {
```

Comment:
nit: Can we just use

```
#if BUILDFLAG(USE_STARBOARD_MEDIA)
    // TODO(b/326827007): Revisit renderer to support secondary videos.
    if (media::IsStarboardRendererEnabled()) {
      factory_selector->AddFactory(
          RendererType::kStarboard,
          std::make_unique<media::StarboardRendererFactory>(media_log));
      factory_selector->SetBaseRendererType(RendererType::kStarboard);
    }
#endif // BUILDFLAG(USE_STARBOARD_MEDIA)
```

and then fallback to the block below (and remove the #else)?